src/pf-modal/index.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+<head>
+    <title>pf-tooltip example</title>
+    <link rel="stylesheet" href="/dist/css/patternfly.css">
+    <link rel="stylesheet" href="/dist/css/patternfly-additions.css">
+    <link rel="stylesheet" href="/dist/css/patternfly-webcomponents.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/0.7.22/webcomponents.js"></script>
+    <script src="/dist/js/patternfly.js"></script>
+</head>
+<body>
+<div class="container">
+    <div class="page-header">
+      <h1>Modals</h1>
+    </div>
+    <hr>
+    
+    <button class="btn btn-default" id="btn-toggle-modal">Launch demo modal</button>
+    <pf-modal targetSelector="#btn-toggle-modal" backdrop keyboard>
+      <pf-modal-header modalTitle="Modal Title"></pf-modal-header>
+      <pf-modal-body>
+        <form class="form-horizontal">
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="textInput">Field One</label>
+            <div class="col-sm-9">
+              <input type="text" id="textInput" class="form-control"></div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="textInput2">Field Two</label>
+            <div class="col-sm-9">
+              <input type="text" id="textInput2" class="form-control"></div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="textInput3">Field Three</label>
+            <div class="col-sm-9">
+              <input type="text" id="textInput3" class="form-control">
+            </div>
+          </div>
+        </form>
+      </pf-modal-body>
+      <pf-modal-footer></pf-modal-footer>
```

Comment:
I wasn't specifically speaking about the modal example, but something more like listView, toolbar, etc. Click on the buttons in the example below and you'll see the interaction.

http://www.patternfly.org/angular-patternfly/#/api/patternfly.views.directive:pfListView